ar4si.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package ar4si
```

Comment:
Ideally I would prefer the name "attestationResults" both for the package name and also github name.

Yes, one may argue that this name is aligned with what they use in standards body but strictly speaking it is not a very intuitive name and also 4 does not really map to for